docs/introduction.rst

```diff
@@ -56,14 +56,17 @@ platforms. It provides functionality to:
   to execute the :term:`CIM-XML` operations defined in the DMTF specification
   CIM Operations Over HTTP (:term:`DSP0200`).
 
+* Execute specific CIM/XML operations defined in the DMTF CIM Operations Over
+  HTTP (:term:`DSP0200`). See section :ref:`pywbemcli commands to WBEM Operations`_
+
```

Comment:
This repeats the previous paragraph, so it was removed in an earlier PR. Your PR seems to have been based on a version of the docs before that, so it reintroduces the paragraph. I suggest to remove it again.

Reply:
ok

Reply:
Removed sentence in first paragraph